recipe/meta.yaml

```diff
@@ -1,33 +1,33 @@
-{% set version = "23.1.0" %}
-{% set hash = "090070a54572236f8e23263659520a27c31161e595a1c4feaffb70263d86a32e" %}
-{% set build_number = "0" %}
+{% set version = "23.3.0" %}
 
 package:
   name: conda-libmamba-solver
   version: {{ version }}
 
 source:
   url: https://github.com/conda/conda-libmamba-solver/archive/refs/tags/{{ version }}.tar.gz
-  sha256: {{ hash }}
+  sha256: 0c209980d10d2acf3bb070f2a00c157319efc49f313e23992f7c350a37d98a87
+  folder: src/
 
 build:
-  skip: true  # [py<37]
+  skip: true  # [py<38]
   # trigger 1
-  number: {{ build_number }}
-  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{version}}
+  script: {{ PYTHON }} -m pip install src/ -vv --no-deps
 
 requirements:
   host:
     - python
     - pip
-    - wheel
-    - setuptools
-    - flit-core >=3.2,<4
+    - hatchling
+    - hatch-vcs
   run:
     - python
-    - libmambapy >=1.0.0
-    - conda >=22.11.0
-    - importlib-metadata
+    - conda >=23.3.1
+    - libmambapy >=1.4.1
+    - boltons >=23.0.0
 
 test:
   imports:
```

Comment:
No, the version isn't baked into the repo anymore but derived from the version control tag during building, or in case of the pattern used here, assigned by setting the env variable `SETUPTOOLS_SCM_PRETEND_VERSION`. This is following the docs from upstream: https://conda-forge.org/docs/maintainer/knowledge_base.html#using-setuptools-scm